src/Core/Users/user.php

```diff
@@ -0,0 +1,142 @@
+<?php
+namespace Meetup\Login\Core;
+class User implements UserInterface {
+    protected $firstName;
+    protected $lastName;
+    protected $emailid;
+    protected $created;
+    protected $datetime;
+    protected $passwd;
+    /**
+     * Set first name
+     *
+     * @param string $firstName
+     *
+     * @return void
+     */
+    public function setFirstName($firstName) {
+        $this->firstName = $firstName;
+    }
+	 /**
+     * Get first name
+     *
+     * @return string
+     */
+    public function getFirstName() {
+        return $this->firstName;
+    }
+    /**
+     * Set last name
+     *
+     * @param string $lastName
+     *
+     * @return void
+     */
+    public function setLastName($lastName) {
+        $this->lastName = $lastName;
+    }
+     /**
+     * Get last name
+     *
+     * @return string
+     */
+    public function getLastName() {
+        return $this->lastName;
+    }
+    /**
+     * Set email
+     *
+     * @param string $emailid
+     *
+     * @return void
+     */
+    public function setEmail($emailid) {
+        $this->email = $emailid;
+    }
+    /**
+     * Get email
+     *
+     * @return string
+     */
+    public function getEmail() {
+        return $this->email;
+    }
+    /**
+     * Get the ID 
+     *
+     * @return integer
+     */
+    public function getId() {
+        return $this->id;
+    }
+    /**
+     * Set created
+     *
+     * @param \DateTime $created
+     *
+     * @return void
+     */
+    public function setCreated($created) {
```

Comment:
Wrong method name. Its `setCreatedAt` NOT `setCreated`. Please read the interface docs before implementing classes.